docs/developer/introduction.md

```diff
@@ -104,10 +105,36 @@ documented there. If it is _not_ documented there, please ask us - chances are y
 not the only one wondering. Of course, also feel free to challenge the decisions by
 starting a discussion on [GithubDiscussions] or by opening a new [github issue].
 
-## Source Layout
-
-Best practices and layout is documented in README files next to the source files they
-describe. For example, take a look at [api/src/](https://github.com/openkfw/TruBudget/tree/main/api/src) to get started.
+## Code Structure and Source Layout
+
+TruBudget project consists of multiple folders under source. These folders contain code and configuration for the different layers of services or utilities. Each folder has its own README file that describes the service in more detail. For example, README file under api folder [api/README.md](https://github.com/openkfw/TruBudget/blob/main/api/README.md) describes individual environment variables that are used by the api service, how to run api as standalone and how to generate code documentation. 
+
+We strongly advise you to read the README files in the relevant folders of the services/features you will be working on. 
+
+List of services/utilities under TruBudget Project:
+
+(Services marked with * represents the core TruBudget services, which are the minimum requirements to run TruBudget.)
```

Comment:
I would first enumerate all TruBudget services in a section and then add another section with additional components/utilities (not sure about the section name)